azure-toolkit-ide-libs/azure-toolkit-ide-common-lib/src/main/java/com/microsoft/azure/toolkit/ide/common/IActionsContributor.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+package com.microsoft.azure.toolkit.ide.common;
+
+import com.microsoft.azure.toolkit.ide.common.action.AzureActionManager;
+
+public interface IActionsContributor {
+    default void registerActions(AzureActionManager am) {
+    }
+
+    default void registerGroups(AzureActionManager am) {
+    }
+
+    default void registerHandlers(AzureActionManager am) {
```

Comment:
It looks these three methods should belong to its parameter `AzureActionManager`